### ISSUE TITLE
Swagger `Try it out` missing parameters when using `include_router` with a url variable in the prefix

### ISSUE THREAD

**Issue by scotchneat** [tiangolo/fastapi#665]
**Describe the bug**
When a router is added to an app using `include_router`, and the `prefix` parameter contains a url variable (e.g. '/{account}/devices'), the variable is not included as a parameter in the swagger documentation.

**To Reproduce**
1. Create a router using APIRouter()
2. Add the router to the app using `app.include_router` with a prefix.
3. Add a url variable to the prefix
```
app.include_router(
    device_router,
    prefix="/{account}/devices"
)
```
4. Launch the application and browse to /docs
5. Expand the path and click the `Try it out` button

**Expected behavior**
When browsing the swagger docs on http://localhost:8080/docs, I want to be able to use the `Try it out` feature.

**Screenshots**
![image](https://user-images.githubusercontent.com/8887224/67720454-f101f900-f9a1-11e9-830b-ab1670653ba8.png)

**Environment:**
 - OS: [e.g. Linux / macOS]
 - FastAPI Version: 0.38.1

- Python version: 3.7

**Additional context**

**Comment by euri10** [comment]
Didn't test but should you copy the openapi.json output to the swagger editor website, do you get the same issue, this is just to get a sense if the generated schema looks correct

**Comment by dmontagu** [comment]
Yeah this *sounds* like a bug in the openapi schema generation. Probably worth a check to see if updating to 0.42 fixes it.

**Comment by scotchneat** [comment]
I have another app on 0.39.1 that I tried this with and it looks like it is working. 
I'm going to test a couple more cases before closing this.

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

I assume you solved your problem, so thanks for reporting back and closing the issue :+1:
